js/main.js

```diff
@@ -1,141 +0,0 @@
-// API key set globally so that it is easily edited if needed
-const apiKey = '7e8e7a9139e4f09fb253db0d8ddce620';
-
-// Zip value set globally to be edited later by user input
-let zip = ''
-
-
-// Values set globally to be edited later with stateValue()
-let weatherConditions = [
-    {
-        city: '',
-        kTemp: '',
-        fahrenheitTemp: '',
-        celsiusTemp: '',
-        condition: '',
-        icon: '',
-
-    }
-]
```

Comment:
Would be good to set these with "typehinted" values. If they are all strings this is good. But just pointing that out.

Reply:
Looks like you might be hoping to extend this in the future with multiple zips saved so I like that.